src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -1,4 +1,9 @@
 {
+  "stsjorbsmod:Abjure": {
+    "NAME": "Abjure",
+    "DESCRIPTION": "Draw all Spirit Shields from your draw pile, discard pile and exhaust pile.",
```

Comment:
I'm not sure `Draw` is the right word to use here, since I don't think this would actually trigger "on draw" effects from other powers/relics